resources/views/feed.blade.php

```diff
@@ -0,0 +1,24 @@
+<?=
+'<?xml version="1.0" encoding="UTF-8"?>' . PHP_EOL
+?>
+<rss version="2.0">
+    <channel>
+        <title><![CDATA[ {{ url("/") }} ]]></title>
+        <link><![CDATA[ {{ url("/feed") }} ]]></link>
+        <description><![CDATA[ {{ config("canvas.description") }} ]]>
+        </description>
+        <language>{{ app()->getLocale() }}</language>
+        <pubDate>{{ now() }}</pubDate>
+        <lastBuildDate>{{ now() }}</lastBuildDate>
+        @foreach($posts as $post)
+            <item>
+                <title><![CDATA[{{ $post->title }}]]></title>
+                <link>{{ url($post->slug) }}</link>
```

Comment:
Passing in the post slug is good, but we should be using the absolute path to the post - which can vary. Depending if someone wants their publication at: /blog, or /news, etc. 

You could try to use [this config value](https://github.com/austintoddj/canvas/blob/develop/config/canvas.php#L42), but there could still be edge cases.

Reply:
we can have two approaches.

1. we can ask the user to add the static path of the post in the config.
2. we can make the feed view publishable, so if the user has a custom url he can configure the feed accordingly.

or we can do both.

what do you say about that ?

Reply:
> we can ask the user to add the static path of the post in the config

I don't think specifying a static path per post is a good UX. 

> we can make the feed view publishable...configure the feed accordingly

Still a heavier lift than what I'd expect as a user. Essentially, I'd like to see it where you can provide a config value in your .env file - we'd also need to take into account other taxonomies (tags/topics) - and then a switch where you can either enable or disable the feature.